components/edge-identity/backend/apps/default/schema.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS Edge (
+	edge_id INTEGER PRIMARY KEY,
+	name TEXT NOT NULL,
+	namespace TEXT NOT NULL,
+	created_at TEXT NOT NULL,
+	join_token TEXT NOT NULL
```

Comment:
Both for the schema as well as the model please keep in mind that we will need to support different types of edges i.e. ones using "join tokens" to authenticate but also ones that will use different means like pre-shared certs. Thus I'd use a table to store the edge identifier and type plus dedicated tables and specific models for the respective edge type details like the join token.

Reply:
Makes sense. I've added a new `JoinTokenAuth` table that references each `Edge`.